### PR TITLE
Split out chart into smaller charts

### PR DESCRIPTION
**Pull request by fboulnois**
Split out megachart into multiple charts based on Dan's templates

### CODE REVIEW

File: dev/fb/create-secret-manager-secret.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    vals:
+      - kubeSecretKey: datarepopassword
+        path: secret/dsde/datarepo/dev/helm-datarepodb-dev
+        vaultKey: datarepopassword
+      - kubeSecretKey: stairwaypassword
+        path: secret/dsde/datarepo/dev/helm-datarepodb-dev
+        vaultKey: stairwaypassword
+  - secretName: oidc-tls-key
+    vals:
+      - kubeSecretKey: tls.key
```
Comment (smark88) [inline dev/fb/create-secret-manager-secret.yaml:19]: remove this as you are on managed cert

File: dev/fb/oidc-proxy.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  ENABLE_STACKDRIVER: yes
+  FILTER2: AddOutputFilterByType DEFLATE application/json text/plain text/html application/javascript application/x-javascript
+ingress:
+  enabled: true
+  annotations:
+    kubernetes.io/ingress.global-static-ip-name: jade-dev-fb
+  tls:
+    - hosts:
+        - jade-fb.datarepo-dev.broadinstitute.org
+      secretName: oidc-tls-key
```
Comment (smark88) [inline dev/fb/oidc-proxy.yaml:18]:
```suggestion

```

File: dev/fb/oidc-proxy.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  PROXY_URL: http://fb-jade-datarepo-ui.fb:80/
+  PROXY_URL2: http://fb-jade-datarepo-api.fb:8080/api
+  PROXY_URL3: http://fb-jade-datarepo-api.fb:8080/register
+  LOG_LEVEL: debug
+  SERVER_NAME: jade.datarepo-dev.broadinstitute.org
+  REMOTE_USER_CLAIM: sub
+  ENABLE_STACKDRIVER: yes
+  FILTER2: AddOutputFilterByType DEFLATE application/json text/plain text/html application/javascript application/x-javascript
+ingress:
+  enabled: true
```
Comment (smark88) [inline dev/fb/oidc-proxy.yaml:12]:
```suggestion
  enabled: true
   domainName: jade-fb.datarepo-dev.broadinstitute.org
```

File: dev/fb/fbDeployment.yaml [hunk 4]
```diff
@@ -69,7 +69,7 @@ oidc-proxy:
     SERVER_NAME: jade.datarepo-dev.broadinstitute.org
     REMOTE_USER_CLAIM: sub
     ENABLE_STACKDRIVER: yes
-    FILTER2: AddOutputFilterByType DEFLATE application/json text/plain text/html application/javascript application/x-javascript
+    FILTER2: AmsOutputFilterByType DEFLATE application/json text/plain text/html application/javascript application/x-javascript
```
Comment (smark88) [inline dev/fb/fbDeployment.yaml:72]: revert this

Comment (fboulnois) [reply dev/fb/fbDeployment.yaml:72]: haha yeah

Comment (smark88) [inline dev/fb/fbDeployment.yaml:72]:
```suggestion
    FILTER2: AddOutputFilterByType DEFLATE application/json text/plain text/html application/javascript application/x-javascript
```

**Comment by smark88** [comment]
yamlint syntax https://github.com/broadinstitute/datarepo-helm-definitions/runs/731615803#step:4:68

**Pull request merged**